src/hooks/useDeck.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+/**
+ * Exposes DeckContext for use in Deck component and its children
+ */
+
+export const DeckContext = React.createContext();
+
+function useDeck(initialState) {
```

Comment:
We don't have to do this now, but eventually we'll want some logic for handling the end of the slides. Maybe we could add another argument `numSlides`. Then we could do `{ currentSlide: (state.currentSlide + 1) % 3 }` to wrap around to the first slide. Just a thought 🤷‍♀

Reply:
We're thinking it could be good to have a `loop` (or `noLoop`, depending on our default) prop for the deck, where people can decide whether they want it to loop back to the start or to just prevent any change

Reply:
Sounds good to me